config/initializers/devise.rb

```diff
@@ -18,7 +18,7 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
+  config.mailer_sender = "change-password@#{ENV['MAILGUN_DOMAIN']}"
```

Comment:
Since this gets used for confirmation emails too, we should probably call it account@... to avoid confusion from users.

Reply:
Looks like the confirmations PR takes this on